### PR TITLE
Online weight update [WIP]

### PR DESCRIPTION
**Pull request by zhaochenyang20**
## Checklist

- [ ] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/references/contributor_guide.md).
- [ ] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: test/srt/stderr.txt
Comment (zhyncs) [inline test/srt/stderr.txt:1]: Delete this, it's useless.

File: test/srt/stdout.txt
Comment (zhyncs) [inline test/srt/stdout.txt:1]: Delete this, it's useless.

File: test/srt/test_init_parameter_update_group.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+)
+
+mp.set_start_method("spawn", force=True)
+
+
+class TestParameterUpdateGroup(unittest.TestCase):
+    @classmethod
+    def init_process(cls, rank, world_size, base_url, model_name, server_pid):
+        try:
+            # 设置分布式环境
```
Comment (zhyncs) [inline test/srt/test_init_parameter_update_group.py:26]: Do not use Chinese annotations.

**Review by zhyncs — CHANGES_REQUESTED** [review]

**Comment by merrymercy** [comment]
You can rebase and add your 2-gpu test here https://github.com/sgl-project/sglang/blob/fa271613809bc5d901c0e864c4f9b9d3d3a101bd/.github/workflows/pr-test.yml#L106-L109

**Pull request closed without merging**